### PR TITLE
Visually separated toolbar items with an empty space.

### LINKED ISSUE #11: Missing gaps between children of toolbar
A followup of https://github.com/ckeditor/ckeditor5-ui-default/issues/12.

When added to the toolbar, buttons, icons and plain text have no spacing in between, which is visible since the manual theme test has been ported to MVC architecture in [#12](https://github.com/ckeditor/ckeditor5-ui-default/issues/12). It has been patched in MT's `<style>` definition but it must be fixed globally for the entire UI framework. The thing was overlooked because initial designs assumed that the gaps are neglected in the editor toolbar (within `.ck-editor` namespace).

### PR DESCRIPTION
**Pull request by oleq**
Fixes #11.

### CODE REVIEW

File: theme/components/toolbar.scss [hunk 1]
```diff
@@ -16,12 +16,16 @@
...
 	}
 
 	&-newline {
 		display: block;
 		clear: left;
 		height: ck-spacing( 'small' );
 	}
+
+	// (#11) Separate toolbar items.
+	& > * + * {
```
Comment (Reinmar) [inline theme/components/toolbar.scss]: Is it some kind of emoji?

File: theme/components/toolbar.scss [hunk 2]
```diff
@@ -16,12 +16,16 @@
...
+		margin-left: ck-spacing( 'small' );
```
Comment (Reinmar) [inline theme/components/toolbar.scss]: Won't that look bad if items exceeds the available width? The one that appeared in a new line will have left margin.

Comment (Reinmar) [inline theme/components/toolbar.scss]: screen shot 2016-07-05 at 16 57 52

File: theme/components/toolbar.scss (hunk 1 above)
Comment (oleq) [inline theme/components/toolbar.scss]: Yes. It translates to _"You're gonna have troubles maintaining it"_.

**Pull request merged**